src/WebJobs.Script/Rpc/LanguageWorkerConstants.cs

```diff
@@ -10,9 +10,14 @@ public static class LanguageWorkerConstants
         public const string NodeLanguageWorkerName = "node";
         public const string JavaLanguageWorkerName = "java";
         public const string PowerShellLanguageWorkerName = "powershell";
+        public const string PythonLanguageWorkerName = "python";
         public const string WorkerConfigFileName = "worker.config.json";
         public const string DefaultWorkersDirectoryName = "workers";
 
+        // Operating system
+        public const string WindowsOperatingSystemName = "windows";
```

Comment:
Added a comment below, but no need to define our own OS constants. Use `OSPlatform` instead.